### PR TITLE
rlottie: Supports the Transform Position property.

### PR DESCRIPTION
**Pull request by JSUYA**
Use setValue\<rlottie::Property::TrPosition\> and return rlottie::Point type.
Example) example/demo

### CODE REVIEW

File: src/lottie/lottieitem.cpp [hunk 1]
```diff
@@ -830,16 +830,18 @@ DrawableList LOTShapeLayerItem::renderList()
 bool LOTContentGroupItem::resolveKeyPath(LOTKeyPath &keyPath, uint depth,
                                          LOTVariant &value)
 {
-    if (!keyPath.matches(name(), depth)) {
```
Comment (hermet) [inline src/lottie/lottieitem.cpp]: I think you should make a separate patch for this change.

File: src/binding/c/lottieanimation_capi.cpp [hunk 2]
```diff
@@ -204,8 +204,13 @@ lottie_animation_property_override(Lottie_Animation_S *animation,
         animation->mAnimation->setValue<rlottie::Property::StrokeWidth>(keypath, (float)width);
         break;
     }
+    case LOTTIE_ANIMATION_PROPERTY_TR_POSITION: {
+        double x = va_arg(prop, double);
```
Comment (hermet) [inline src/binding/c/lottieanimation_capi.cpp]: We may need to change double argument type to float as input. 
Not this one, that must be an additional patch.

File: src/lottie/lottieitem.h [hunk 3]
```diff
@@ -348,6 +348,8 @@ class LOTContentGroupItem: public LOTContentItem
    LOTGroupData                                  *mData{nullptr};
    std::vector<LOTContentItem*>   mContents;
    VMatrix                                        mMatrix;
+private:
+   LOTProxyModel<LOTGroupData> mModel{nullptr};
```
Comment (hermet) [inline src/lottie/lottieitem.h]: How this non pointer instance can be nullptr? Is it correct?

File: src/lottie/lottieproxymodel.h [hunk 4]
```diff
@@ -334,4 +340,22 @@ class LOTProxyModel<LOTFillData>
...
 };
 
+template <>
+class LOTProxyModel<LOTGroupData>
+{
+public:
+    LOTProxyModel(LOTGroupData *model): _modelData(model) {}
+    LOTFilter& filter() {return mFilter;}
+    const char* name() const {return _modelData->name();}
+    VPointF point(int frame) const
```
Comment (hermet) [inline src/lottie/lottieproxymodel.h]: point is a good name? how about position?

Comment (JSUYA) [reply src/lottie/lottieproxymodel.h]: I will change to position

File: src/lottie/lottieitem.cpp (hunk 1 above)
Comment (JSUYA) [reply src/lottie/lottieitem.cpp]: I dont understand. 
This is change that occurred when the mModel was added.

File: src/binding/c/lottieanimation_capi.cpp (hunk 2 above)
Comment (JSUYA) [reply src/binding/c/lottieanimation_capi.cpp]: i will make another patch.

File: src/lottie/lottieitem.h (hunk 3 above)
Comment (JSUYA) [reply src/lottie/lottieitem.h]: As far as i know. this is correct.
please see lottieproxymodel.h 244 line

File: src/lottie/lottieitem.cpp (hunk 1 above)
Comment (hermet) [reply src/lottie/lottieitem.cpp]: ok, got it.

File: src/lottie/lottieitem.h (hunk 3 above)
Comment (hermet) [reply src/lottie/lottieitem.h]: lottieproxymodel.h 244 line => see what?
244         return LottieColor(col.r(), col.g(), col.b());

Comment (JSUYA) [reply src/lottie/lottieitem.h]: I update code. Thank you for your review.
